src/js/actions/deviceActions.js

```diff
@@ -590,6 +592,28 @@ export const getSessionDetails = (sessionId, deviceId, userId, startDate, endDat
   );
 };
 
+export const deviceFileDownload = (deviceId, path) => () => GeneralApi.post(`${deviceConnect}/devices/${deviceId}/download`, { path });
+
+export const deviceFileUpload = (deviceId, path, file) => dispatch => {
+  var formData = new FormData();
+  formData.append('file', file);
+  formData.append('path', path);
```

Comment:
File must be the last field in the multipart form-data:

```suggestion
  formData.append('path', path);
  formData.append('file', file);
```

Reply:
🤦 sorry - is now fixed!